build.rs

```diff
@@ -67,13 +67,34 @@ fn gen_cxxbridge() -> Result<()> {
     let out_path = PathBuf::from(out_path).join("gen");
     let src_out_path = PathBuf::from(&out_path).join("src");
     let header_out_path = PathBuf::from(&out_path).join("include").join("rust");
+    let header_out_path_sapling = header_out_path.join("sapling");
+
+    // Copy the files we need to generate the bindings
+    fs::copy(
+        "depend/zcash/src/rust/src/sapling/spec.rs",
+        "depend/zcash/src/rust/src/spec.rs",
+    )
+    .unwrap();
+    fs::copy(
+        "depend/zcash/src/rust/src/sapling/zip32.rs",
+        "depend/zcash/src/rust/src/zip32.rs",
+    )
+    .unwrap();
```

Comment:
If we don't move the files inside sapling compilation will fail.

Reply:
This is probably ok, but Conrado might be able to help change the import paths so this isn't needed.

Reply:
Yea, we can't do it this way as we are only supposed to change things in the `out` but nothing in the `depend/zcash`. We could of couse, disable the verification but i will like to avoid it.